notebooks/utils/model_upcast_utils.py

```diff
@@ -1,197 +1,329 @@
+from collections import defaultdict, OrderedDict
+from dataclasses import dataclass
+from typing import List, Dict, Union
+
 import numpy as np
-import openvino as ov
-from openvino._pyopenvino import Node
+from openvino.runtime import Model, Node
 from openvino.runtime.op import Parameter, Constant
-from openvino.runtime.opset12 import matmul, convolution
+import openvino.runtime.opset12 as opset
 from openvino.runtime.utils.types import get_element_type
-from typing import List, Dict, Union, Tuple
 
-ops_to_track_map = {
-    'Convolution': convolution,
-    'MatMul': matmul,
+import openvino as ov
+from tqdm.auto import tqdm
+
+OPERATION_TYPE_MAP = {
+    "MatMul": opset.matmul,
+    "Convolution": opset.convolution,
+    # "Softmax": opset.softmax,
```

Comment:
does other ops accidentally commented?

Reply:
These are actually not required for FP16 calibration, because only MatMuls and Convs are observed to have an impact. 

The commented operations here were needed during BF16 calibration experiments. But as stated in the PR description and a warning inside the upcasting function, BF16 calibration does not currently provide any improvement. That's why I commented them out. 

But I'll remove these lines altogether to avoid the confusion.